packages/doenetml-worker-rust/lib-doenetml-core/src/core/components/doenet/text_input.rs

```diff
@@ -191,6 +171,14 @@ struct ValueStateVarInterface {
     dependency_values: ValueDependencies,
 }
 
+impl ValueStateVarInterface {
```

Comment:
I forget...why is there `Interface` attached to these names?

Reply:
One piece of `StateVar<T>` is the "interface", which is the trait object defining the functions to set dependencies, calculate value, etc. The rest of the state variable is typed. Not at all stuck on the term "interface".